Permit enums to begin with a digit (#233)
FamilySearch/GEDCOM: specification/gedcom-02-datatypes.md

## Enumeration

An enumeration is a selection from a set of options.
They are represented as a string matching the same production as a tag,

> with the additional permission that standard enumerations may begin with a digit.

Author: ```suggestion
with the additional permission that standard enumerations may be integers.
```

```abnf

> stdEnum = stdTag / digit *tagchar

Author: ```suggestion
stdEnum = stdTag / Integer
```

Author: Less permissive option
